src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/RemoveUnusedLocals.Fixer.cs

```diff
@@ -1,27 +1,174 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Composition;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis.FindSymbols;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeActions;
 
 namespace Microsoft.CodeQuality.Analyzers.Maintainability
 {
     /// <summary>
     /// CA1804: Remove unused locals
     /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic, Name = AvoidUnusedPrivateFieldsAnalyzer.RuleId), Shared]
```

Comment:
Export attribute should only be applied on concrete non-abstract types. Can you move it to C# and VB sub-types of the fixer?